conf/modules/stereocam_follow_egg.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="stereocam_follow_egg" dir="stereocam/follow_egg">
+  <doc>
+    <description>follows the egg next to the mavlab</description>
```

Comment:
I think it would help to have a bit more descriptive text here. Especially which sensor is used (dedicated sensor, on-board Parrot camera, ...).
Same thing for the follow_me module.